[260] Create unit test to HabitStatisticController

ChernenkoVitaliy opened the pull request:
Created unit tests for HabitStatisticController;
Added method updateHabitStatisticDto for creating UpdateHabitStatisticDto object instance into ModelUtils.class;

ChernenkoVitaliy committed:
Created unit tests fot HabitStatisticController;
Added method updateHabitStatisticDto for creating UpdateHabitStatisticDto object instance into ModelUtils.class

ZenukD reviewed (approved):
You use a variable UserVO in four methods, perhaps it would be convenient to take it out separately

ChernenkoVitaliy committed:
Repeated UserVO has been taken out from four methods and added as the class level variable;

VitaliiChernenko commented:
> You use a variable UserVO in four methods, perhaps it would be convenient to take it out separately

Thanks, fixed.

ChernenkoVitaliy committed:
Removed passing userVO as parameter, use .principal() instead;
added final fields for ids to avoid duplication;
added setCustomArgumentResolvers for correct .principal() calling;
renamed some methods according to naming convention;

vulook reviewed (approved):
LGTM
